evaluator/src/test/kotlin/PackageRuleTest.kt

```diff
@@ -118,7 +121,8 @@ class PackageRuleTest : WordSpec() {
 
             "return false if the license is not an SPDX license" {
                 PackageRule(ruleSet, "test", packageWithoutLicense, emptyList(), emptyList()).apply {
-                    val licenseRule = LicenseRule("test", "invalid", LicenseSource.DECLARED, emptyMap())
+                    val licenseRule =
+                        LicenseRule("test", SpdxLicenseIdExpression("invalid"), LicenseSource.DECLARED, emptyMap())
```

Comment:
Nit: Formatting to the above should be aligned.

Reply:
Done.